scripts/initReport.mjs

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+
+/**
+ * This script creates a new report file with the current date on a new branch.
+ */
+
+import fs from 'fs'
+import { fileURLToPath } from 'url'
+import { execSync } from 'child_process'
+
+const REPORTS_DIRECTORY = 'content/reports'
+const REPORT_TYPE = 'contributors'
+const PULLS_URL =
+  'https://api.github.com/search/issues?q=is:pr+user:TOSIT-IO+is:open+sort:updated-asc'
+
+function formatCurrentDate(date) {
+  const year = date.getFullYear().toString().slice(-2)
+  const month = (date.getMonth() + 1).toString().padStart(2, '0')
+  const day = date.getDate().toString().padStart(2, '0')
+  return `${year}${month}${day}`
+}
+
+function checkoutNewBranch(branchName) {
+  try {
+    const currentBranch = execSync('git branch --show-current')
+      .toString()
+      .trim()
+
```

Comment:
Never place empty lines inside a function, use a comment if you mean to express an idea with an empty space. Also apply to many lines below.